views/sign_up.erb

```diff
@@ -5,19 +5,22 @@
     Create a username
   </label>
   <br>
-  <input type='text' name='username'>
+  <input type='text' required name='username' oninvalid="this.setCustomValidity('Enter your name here')"
```

Comment:
Could you change this to `Enter your username here`? Just to be specific that it doesn't have to be your name